src/auction.js

```diff
@@ -236,7 +236,7 @@ export const addBidResponse = createHook('asyncSeries', function(adUnitCode, bid
 
```

Comment:
@snapwich Not sure if I did something wrong when checking this branch out locally but I see one remaining reference to `usePrebidCache` in this file on line 155. Everything else lgtm

Reply:
No you are correct.  It looks like Jaimin added a new call to `usePrebidCache` after the creation of this pull-request which I got with the latest merge update.  Just fixed and updated, as well as removed a call to it in one of the bid adapters.